src/common/request.factory.js

```diff
@@ -45,6 +46,12 @@ export default /* @ngInject */ function (
     return this;
   }
 
+  ApiRequest.prototype.setApiVersion = function (apiVersion) {
+    const clone = this.clone();
+    clone.apiOptions.apiVersion = angular.isDefined(apiVersion) ? apiVersion : 'apiv6';
```

Comment:
**question**: shouldn't we use `angular.isString` here ?

Reply:
This is done, thx